packages/v2/gui/src/sak/meny/marker-behandling/MenyMarkerBehandling.tsx

```diff
@@ -1,35 +1,24 @@
 /* eslint-disable arrow-body-style */
-import type { MerknadDto } from '@k9-sak-web/backend/k9sak/generated';
+import { K9SakClientContext } from '@k9-sak-web/gui/app/K9SakClientContext.js';
+import { useContext } from 'react';
 import MarkerBehandlingModal from './components/MarkerBehandlingModal';
+import MarkerBehandlingBackendClient from './MarkerBehandlingBackendClient';
 
 interface OwnProps {
   lukkModal: () => void;
-  brukHastekøMarkering?: boolean;
-  brukVanskeligKøMarkering?: boolean;
-  markerBehandling: (values: any) => Promise<any>;
   behandlingUuid: string;
-  merknaderFraLos: MerknadDto;
   erVeileder?: boolean;
 }
 
-const MenyMarkerBehandlingV2 = ({
-  lukkModal,
-  brukHastekøMarkering,
-  brukVanskeligKøMarkering,
-  markerBehandling,
-  behandlingUuid,
-  merknaderFraLos,
-  erVeileder,
-}: OwnProps) => {
+const MenyMarkerBehandlingV2 = ({ lukkModal, behandlingUuid, erVeileder }: OwnProps) => {
+  const k9SakClient = useContext(K9SakClientContext);
+  const markerBehandlingBackendClient = new MarkerBehandlingBackendClient(k9SakClient);
```

Comment:
```suggestion
    const markerBehandlingBackendClient = useMemo(
        () => new MarkerBehandlingBackendClient(k9SakClient),
        [k9SakClient]
   );
```
`markerBehandlingBackendClient` genereres på hver render, kanskje den burde "memoiseres"?

Reply:
Nei, det er ikke noe problem. La inn en console.log rett over i samme fil, og den kjørte bare en gang selv når jeg klikket rundt i modalen. Kan optimalisere dersom det blir behov.